src/digitalIdentity-backend/src/main/java/didentity/amos/digitalIdentity/services/LissiApiService.java

```diff
@@ -104,10 +104,14 @@ public ResponseEntity<String> provideExistingSchemas(String activeState, String
         return handleResponse(response);
     }
 
+    @SuppressWarnings("unchecked") // TODO: if someone wants to bother with generic arrays, feel free :)
```

Comment:
It is useless because this declaration is already done for the whole class. So you can delete this line 107.

Reply:
it's done